### PR TITLE
Update website to lantern theme

### PR DESCRIPTION
**Pull request by debanjum**
- Use new lantern logo in website favicon, header logo
- Use new hero and footer section artwork
- Update website colors to yellow from blue to match new theme
- Center align logo and navigation links in header

### CODE REVIEW

File: src/NavBar.tsx [hunk 1]
```diff
@@ -7,27 +7,24 @@ export default function NavBar() {
...
-                            </Button>
-                            <Button type="text" shape="default" style={{ borderRadius: '4px' }}>
-                                <Link className="navLinks" to="/about">About</Link>
-                            </Button>
-                            <Button type="text" shape="default" style={{ borderRadius: '4px' }}>
-                                <Link className="navLinks" to="/waitlist">Waitlist</Link>
-                            </Button>
-                        </li>
-                    </ul>
+                    <div></div>
```
Comment (sabaimran) [inline src/NavBar.tsx:15]: Is this empty div required?

File: src/NavBar.tsx [hunk 2]
```diff
@@ -7,27 +7,24 @@ export default function NavBar() {
...
+                    <div className="centeredLinks">
+                        <Button type="primary" size="large" shape="default" style={{borderRadius: '4px'}}>
+                            <Link className="navLinks" to="https://github.com/debanjum/khoj#Setup">Install</Link>
+                        </Button>
+                        <Button type="text" size="large" shape="default" style={{borderRadius: '4px', border: '1px solid #000', backgroundColor: '#f9f5de'}}>
+                            <Link className="navLinks" to="/about">About</Link>
+                        </Button>
+                        <Button type="text" size="large" shape="default" style={{borderRadius: '4px', border: '1px solid #000', backgroundColor: '#f9f5de'}}>
+                            <Link className="navLinks" to="/waitlist">Waitlist</Link>
+                        </Button>
```
Comment (sabaimran) [inline src/NavBar.tsx:25]: Is it possible to drive these styles through css?

File: src/styles/colors.css [hunk 3]
```diff
@@ -1,7 +1,7 @@
 :root {
     --main-background: #282c34;
     --dark-water: #354764;
-    --calm-water: #eaf8f9;
+    --mild-sun: #f9f5de;
```
Comment (sabaimran) [inline src/styles/colors.css:4]: nice!

File: src/NavBar.tsx (hunk 1 above)
Comment (debanjum) [reply src/NavBar.tsx:15]: Yes, for spacing the navigation links to center using css grids

File: src/NavBar.tsx (hunk 2 above)
Comment (debanjum) [reply src/NavBar.tsx:25]: Yeah, I tried that but I couldn't figure out a way to style the antd component library buttons with css styling. It gets overridden by the antd styling unless I add the styling to the element directly (as that takes highest precedence)

**Pull request merged**